tests/app/views/contexts/summary/test_question.py

```diff
@@ -268,6 +268,43 @@ def test_checkbox_answer_with_detail_answer_returns_the_value(self):
         self.assertEqual(len(question.answers[0]['value']), 2)
         self.assertEqual(question.answers[0]['value'][1].detail_answer_value, 'Test')
 
+    def test_checkbox_answer_with_numeric_detail_answer_returns_the_value(self):
+        # Given
+        self.answer_store.add_or_update(
+            Answer(answer_id='answer_1', value=['Light Side', 'Other'])
```

Comment:
This doesn't need to be changed, but the question doesn't make a great deal of sense, Light side or 1. If there is no better question then happy for it to stay as is

Reply:
This question is just copied from the test before it. Don't think it matters too much what the question is for the test but happy to change if people think it needs to make more sense

Reply:
Yeah, it would be better if this did make sense I think. Helps people who are looking at the test to see the use case. Not hugely important though.